### ISSUE TITLE
Multimaterialsupport like Filamentmanager

### ISSUE THREAD

**Issue by BenBetta** [OllisGit/OctoPrint-SpoolManager#153]
It would be nice to choose the filament for all 5 positions in my mmu equipment. So on automatical or manual tool change also the counter for the filament changes to get accurate information about used filament on each spool.

**Comment by codingcatgirl** [comment]
This is a duplicate of #46, which is also stated in the README ;)

**Comment by disconn3ct** [comment]
#46 is closed though?

**Comment by codingcatgirl** [comment]
Yeah, thanks to stalebot. But the last comment should explain what's going on?

**Comment by codingcatgirl** [comment]
I was waiting for the feature for so long that i just wrote it myself :D Here's the pull request, it's fully running on my branch: #161

**Comment by OllisGit** [comment]
Hi,
in the latest release V1.4.0 the PR from @codingcatgirl is now included. It looks really nice. Thanks again!!! 

Please test and give me a feedback.
Olli
